test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestJcmd.java

```diff
@@ -172,12 +224,20 @@ protected void setUp(Method testMethod) {
 	@BeforeSuite
 	protected void setupSuite() {
 		assertTrue(PlatformInfo.isOpenJ9(), "This test is valid only on OpenJ9"); //$NON-NLS-1$
+		userDir = new File(System.getProperty("user.dir")); //$NON-NLS-1$
 		getJdkUtilityPath(JCMD_COMMAND);
 		commandExpectedOutputs = new HashMap<>();
 		commandExpectedOutputs.put(HELP_COMMAND, THREAD_PRINT);
```

Comment:
This is not part of this PR however it seems wrong.

Reply:
This is intended: the test needs to know the current working directory.

Reply:
`new File(".")` is a much simpler way to refer to the current directory.

Reply:
Not sure about the response. My question was about `commandExpectedOutputs.put(HELP_COMMAND, THREAD_PRINT);`, why is `THREAD_PRINT` the expected outputs of `HELP_COMMAND`?

Reply:
> why is `THREAD_PRINT` the expected outputs of `HELP_COMMAND`?

`THREAD_PRINT` is _part_ of the expected output of `HELP_COMMAND`.  I chose it
as a representative sample of the output.